jobs/v4/src/main/java/com/example/jobs/JobSearchCreateJob.java

```diff
@@ -22,6 +22,8 @@
 import com.google.cloud.talent.v4beta1.Job;
 import com.google.cloud.talent.v4beta1.JobServiceClient;
 import com.google.cloud.talent.v4beta1.TenantName;
+import com.google.protobuf.Timestamp;
+import com.google.protobuf.util.Timestamps;
```

Comment:
This wouldn't be my first choice.  https://docs.oracle.com/javase/8/docs/api/java/sql/Timestamp.html would be better.

Reply:
Okay I will change it to sql Timestamp

Reply:
Looks like method that sets expiration date specifically takes this Timestamp:
```
setPostingExpireTime(com.google.protobuf.Timestamp)
```

Since it is discouraged, I will take this out and just comment it about 30 days expiration. Also, it is also mentioned in the docs, https://cloud.google.com/talent-solution/job-search/docs/jobs